roles/create_vms/tasks/main.yml

```diff
@@ -4,8 +4,8 @@
   set_fact:
     kvm_nodes: "{{ kvm_nodes | default([]) + [{
         'name': item,
+        'uuid': hostvars[item]['uuid'] | default(false),
```

Comment:
I think a more intuitive default value would be an empty string. We should not mix different types (uuid string, boolean) for the same variable (`kvm_nodes[*]['uuid']`).